core/src/main/resources/lib/form/select/select.js

```diff
@@ -98,4 +103,4 @@ Behaviour.specify("SELECT.select", 'select', 1000, function(e) {
                 }
             });
         });
-});
+});
```

Comment:
```suggestion
});

```

Reply:
Is there linting in place to guarantee newline at end of file?

Reply:
👍 @oleg-nenashev